tests/test-cmd/test.run.js

```diff
@@ -10,7 +10,7 @@ import {onlyInstancesOf, WebExtError, RemoteTempInstallNotSupported}
 import run, {
   defaultFirefoxClient, defaultWatcherCreator, defaultReloadStrategy,
 } from '../../src/cmd/run';
-import * as firefox from '../../src/firefox';
+import * as firefoxApp from '../../src/firefox';
```

Comment:
@shubheksha how about renaming the above to `defaultFirefox`, like it is named in most of the sources?
it would make easier to guess that they are the same thing.

(to be completely honest, now it would be probably better to name it `defaultFirefoxApp`, here and in "src/cmd/run.js", to make it clear that it is the default implementation of what we usually call `firefoxApp`)